views/components/BarGraphTooltip/index.jsx

```diff
@@ -2,31 +2,28 @@ import React from 'react'
 import { TOTAL_LABEL } from '../../../constants'
 import { isTargetShown, siteTooltipContent } from './helpers'
 
-const BarGraphTooltip = ({ classes, active, payload, label, studyTotals }) => {
+const BarGraphTooltip = ({ active, payload, label, studyTotals }) => {
   if (!active || !payload?.length) {
     return null
   }
 
   return (
-    <div className={classes.tooltipContainer}>
-      <div className={classes.tooltipHeaderRow}>
-        <div className={classes.tooltipLabelColumn}>{label}</div>
-        <div className={classes.tooltipValueColumn}>
-          {isTargetShown(payload) ? 'Value / Target' : 'Value'}
-        </div>
+    <div>
+      <div>
+        <div>{label}</div>
+        <div>{isTargetShown(payload) ? 'Value / Target' : 'Value'}</div>
       </div>
       {siteTooltipContent(payload, studyTotals[label]).map(
         ({ labelColumn, valueColumn }) => {
           return (
             <div
-              className={classes.tooltipContentRow}
```

Comment:
unrelated to this change but I think we need a `key` prop here.

Reply:
[465c369](https://github.com/AMP-SCZ/dpdash/pull/458/commits/465c36952e0f8176957c65d89dfea1c6b86ca296)